<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>478</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>551</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="mainLayout" stretch="0">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <property name="margin">
      <number>5</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBoxBottom">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>121</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>800</width>
         <height>121</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <stylestrategy>PreferDefault</stylestrategy>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="title">
        <string>Game state</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>493</width>
          <height>107</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout" rowstretch="0">
         <property name="leftMargin">
          <number>5</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>6</number>
         </property>
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="topMargin">
            <number>5</number>
           </property>
           <property name="bottomMargin">
            <number>5</number>
           </property>
           <item>
            <widget class="QLabel" name="labelAngle">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>50</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string>Gun 
 angle:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="minimumSize">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
             <property name="minimum">
              <number>0</number>
             </property>
             <property name="maximum">
              <number>95</number>
             </property>
             <property name="value">
              <number>45</number>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
             <property name="textVisible">
              <bool>true</bool>
             </property>
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="bottomMargin">
            <number>2</number>
           </property>
           <item>
            <widget class="QLabel" name="labelPower">
             <property name="minimumSize">
              <size>
               <width>50</width>
               <height>20</height>
              </size>
             </property>
             <property name="text">
              <string>Gun 
 Power:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdNumber">
             <property name="minimumSize">
              <size>
               <width>40</width>
               <height>40</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="smallDecimalPoint">
              <bool>false</bool>
             </property>
             <property name="numDigits">
              <number>2</number>
             </property>
             <property name="digitCount">
              <number>2</number>
             </property>
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
             <property name="value" stdset="0">
              <double>50.000000000000000</double>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBoxStat">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>270</width>
               <height>0</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>11</pointsize>
              </font>
             </property>
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="title">
              <string>Statistics</string>
             </property>
             <property name="flat">
              <bool>false</bool>
             </property>
             <widget class="QPushButton" name="pushButtonRestart">
              <property name="geometry">
               <rect>
                <x>180</x>
                <y>20</y>
                <width>75</width>
                <height>23</height>
               </rect>
              </property>
              <property name="text">
               <string>Restart</string>
              </property>
             </widget>
             <widget class="QWidget" name="">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>20</y>
                <width>159</width>
                <height>25</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Time remaining:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberTimer">
                 <property name="smallDecimalPoint">
                  <bool>false</bool>
                 </property>
                 <property name="numDigits">
                  <number>3</number>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                 <property name="value" stdset="0">
                  <double>0.000000000000000</double>
                 </property>
                 <property name="intValue" stdset="0">
                  <number>0</number>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="">
              <property name="geometry">
               <rect>
                <x>10</x>
                <y>50</y>
                <width>236</width>
                <height>25</height>
               </rect>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Hits:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberHitsCount">
                 <property name="digitCount">
                  <number>3</number>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Shot count:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLCDNumber" name="lcdNumberShotsCount">
                 <property name="digitCount">
                  <number>3</number>
                 </property>
                 <property name="segmentStyle">
                  <enum>QLCDNumber::Flat</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGame">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuGame"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
